components/automate-chef-io/content/docs/saml.md

```diff
@@ -42,12 +42,53 @@ Users who sign in via SAML will have a session time of 24 hours before needing t
 
 ## Supported Identity Management Systems
 
+- [Azure AD]({{< relref "#azure-ad" >}})
 - Office365
 - OKTA
 - OneLogin
 - Ping
 - Tivoli Federated Identity Manager
 
+### Azure AD
+
+Using Azure AD as an SAML IdP requires specific configuration for both Azure AD and Chef Automate.
+
+{{< info >}}
+The signing certificate used for Chef Automate's SAML integration with Azure AD requires manual management.
+Signing key rotation is not done automatically.
+{{< /info >}}
+
+First, set up your configuration within Azure AD.
```

Comment:
@srenatus I would think we could put your official documentation link towards the beginning on either line 61 or 62? Either line may have to be reworded. I just realized both sentences begin with "set up."

Reply:
Linked something :) Not sure about the wording there, though -- please have a look.